client/landing/stepper/declarative-flow/internals/steps-repository/difm-starting-point/style.scss

```diff
@@ -1,6 +1,13 @@
 @import "@wordpress/base-styles/breakpoints";
 @import "@wordpress/base-styles/mixins";
 
+.difm-starting-point {
+	.help-center-step-button__button {
+		font-size: 0.875rem;
+		font-weight: 500;
+	}
+}
```

Comment:
Not great, but I made a note to improve this when the step-container-v2 thing goes through